concent_api/core/views.py

```diff
@@ -884,3 +934,7 @@ def tmp_is_provider_account_status_positive(request):
         return bool(request.META['HTTP_TEMPORARY_ACCOUNT_FUNDS'])
     except KeyError:
```

Comment:
Don't catch `KeyError`. use `in` operator instead.

Reply:
This whole function was not added by me, but I made the fix.